web/cobrands/sass/_base.scss

```diff
@@ -2823,6 +2823,13 @@ $nicetable-hover-background: rgba($primary, 0.15) !default;
     }
 }
 
+.emergency-message {
+  border: 1px solid #525252;
+  padding: 2em;
```

Comment:
This padding and font-size is quite large on narrow screens!

![Screen Shot 2020-11-20 at 09 29 34](https://user-images.githubusercontent.com/739624/99783907-f5500580-2b12-11eb-95d0-1e033409fe7d.png)

Perhaps `font-size: 1em; padding: 1em;` here, and then `font-size: 1.2em; padding: 2em;` in `_layout.scss`?